.buildkite/commands/build.sh

```diff
@@ -1,17 +1,6 @@
 #!/bin/bash -u
 
-echo "--- :swift: Installing Swift Package Manager Dependencies"
-install_swiftpm_dependencies
-
-echo "--- Setup Ruby tooling"
-
-export LANG=en_US.UTF-8
-export LC_ALL=en_US.UTF-8
```

Comment:
Do these need to be added to the `shared_setup.sh` script?

Reply:
They were legacy settings [coming from the previous automation setup](https://github.com/Automattic/pocket-casts-ios-private/commit/cffc29815892339c79d6d9b69947d52fe04d9420). Unfortunately I don't have additional context on the previous automation. 

Those exports triggered some old memories from other projects though, so if I had to guess I'd say they are related to old CocoaPods requirements, e.g. https://github.com/CocoaPods/CocoaPods/issues/6333